### DISCUSSION TITLE
On-line tutorials?

### DISCUSSION THREAD

**Discussion by apopp27** [Ifsttar/I-Simpa#342] [Q&A]
Hi,

I'm an experienced acoustic engineer specialising in room acoustics.  I've used both ODEON and CATT in the past but many years ago!  I've tried I-Simpa before and struggled to work out the basics - I recall the English translation of the instruction manual isn't always clear.  

Please can anyone direct me to any on-line tutorial videos so I can try again?!

Thanks
Apopp

**Comment by Picaut** [comment]
Dear apopp27

The latest document in English is here : https://i-simpa-wiki.readthedocs.io/fr/latest/ 
with three tutorials:
- https://i-simpa-wiki.readthedocs.io/fr/latest/tutorial_teaching_room.html
- https://i-simpa-wiki.readthedocs.io/fr/latest/tutorial_Elmia_hall.html
- https://i-simpa-wiki.readthedocs.io/fr/latest/tutorial_industrial_hall.html

"I recall the English translation of the instruction manual isn't always clear"
Yes, it may be possible. If not clear, could you specify which part, in order for us, to modify/enhance the documentation.

Unfortunately, there is no online video tutorial. 

Best regards.

**Comment by Picaut** [comment]
Additionally, see this comment:
https://github.com/Universite-Gustave-Eiffel/I-Simpa/discussions/340#discussioncomment-3925497
